www/userguide/userguide.md

```diff
@@ -0,0 +1,290 @@
+It is possible to change the order of the runs by clicking on a cell at header
+of the run list table. For example you can sort the run list by the number of
+bugs or the run name.
+
+# <a name="userguide-checker-statistics"></a> Checker statistics
+A statistical overview can be seen under "Checker statistics" panel. In
+this table you can see the number of reports by checkers based on some
+attributes of the report like severity, and report status.
+
+![Checker statistics](images/checker_statistics.png)
+
+# <a name="userguide-analysis-results"></a> Analysis results
+If you select a run at the [list of runs](#userguide-list-of-runs) view, you get
+to this page. This page lists the analysis result for the given run.
+
+![Reports](images/reports.png)
+
+## <a name="userguide-filtering"></a> Filtering
+When opening the bug list view under "All reports" tab or by clicking a specific
+run or by opening "diff view" between two runs then the following filter options
+are available:
+- **Run name**: You can select one or more run names. The result list is
+restricted on the findings in these runs. By selecting a specific run in the
+"runs" view this field is filled by default. In "All reports" tab no run is
+selected in which case the reports from all runs are visible.
+- [**Review status**](#userguide-review-status): You can select the reports with
+the given review status to check only *False positive*, *Unreviewed*, etc.
+reports.
+- [**Detection status**](#userguide-detection-status): You can select the
+reports with the given detection status to check only *Unresolved*, *Resolved*,
+etc. reports.
+- [**Severity**](#userguide-severity-levels): The nature of the bugs is sorted
+in different severity levels. For example a division by zero or a null pointer
+dereference is more serious than an unused variable. By this field you can
+select the reports on the given severity levels.
+- **Run tag**: When runs are stored in update mode (i.e. on the same run name),
+then the specific runs can be tagged in order to be easier to identify them.
+By this field you can select the reports found during a specific run event.
+- **Detection date**: A date interval can also restrict the list of displayed
+bug reports. In this field you can choose the date of detection or fixing.
+- **File**: You can choose a set of files to restrict the list of bug reports.
+- **Checker name**: If you are instrested in specific type of bugs then here you
+can choose them.
+- **Checker message**: The static analysis tools provide a message to indicate
+the reason of a specific bug. This message is also filterable.
+
+In "diff view" instead of "run name" you have the following filter fields:
+- **Baseline**: The runs against which you want to check the difference.
+- **Newcheck**: The runs which you want to compare against the baseline runs.
+- **Diff type**: Here you can set if you'd like to see the bugs which appear
+only in the baseline, newcheck or both.
```

Comment:
I would format baseline and newcheck as `baseline` or as _baseline_ (monospace or italic)